introduction.md

```diff
@@ -30,9 +30,9 @@ to keep your existing scripts (based on previous behavior). Versions will always
 
 The versions start with version 2, since version 1 used to be a non-REST basic API for internal purposes. A notable exception is the *latest* version, which will always point to the most recent version available.
 
-It is possible to select the API version you want to use in:
+You may want to select the API version to call:
```

Comment:
You can change the version of the API used by setting it either within the url or in a header (see below)